iocage/lib/Config/Jail/BaseConfig.py

```diff
@@ -550,7 +581,24 @@ def __getitem__(self, key: str) -> typing.Any:
 
     def __delitem__(self, key: str) -> None:
         """Delete a setting from the configuration."""
-        del self.data[key]
+        self.__deleteitem(key, self.data)
+
+    def __deleteitem(
+        self,
+        key: str,
+        data: typing.Dict[str, typing.Any]
+    ) -> None:
+        if "." in key:
```

Comment:
*mumbles something about nested json*

Reply:
This is about CLI interaction. The internal storage occurs as nested JSON - just the property may be accessed via dot notation.